File: algorithm/src/main/java/baekjoon/p05656/Main.java
package baekjoon.p05656;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int i = 1;

        while (true) {
            int a = sc.nextInt();
            String oper = sc.next();
            int b = sc.nextInt();

            if (oper.equals("E")) {
                break;
            } else {
                System.out.println(String.format("Case %d: %s", i++, solve(oper, a, b)));
            }
        }
    }

    private static boolean solve(String oper, int a, int b) {
        switch (oper) {
            case ">":
                return a > b;
            case ">=":
                return a >= b;
            case "<":
                return a < b;
            case "<=":
                return a <= b;
            case "==":
                return a == b;
            case "!=":
                return a != b;
            default:
                return false;
        }
    }

}

File: algorithm/src/main/java/baekjoon/p05656/Main.kt
package baekjoon.p05656

fun main(args: Array<String>) {
    var i = 1

    while (true) {
        val inputs = readLine()!!.split(" ")
        if (inputs[1] == "E") {
            break
        } else {
            val solve = solve(inputs[1], inputs[0].toInt(), inputs[2].toInt())
            println("Case ${i++}: $solve")
        }
    }
}

private fun solve(oper: String, a: Int, b: Int): Boolean {
    return when (oper) {
        ">" -> a > b
        ">=" -> a >= b
        "<" -> a < b
        "<=" -> a <= b
        "==" -> a == b
        "!=" -> a != b
        else -> false
    }
}